src/LoginPage/LoginFormContainer.jsx

```diff
@@ -31,13 +33,19 @@ export default function LoginFormContainer() {
     dispatch(changeLoginFields({ name, value }));
   }, [dispatch]);
 
+  const handleSubmit = useCallback(() => (
+    isLogin
+      ? dispatch(requestLogin()) : dispatch(requestSignup())
```

Comment:
```suggestion
    isLogin
      ? dispatch(requestLogin())
      : dispatch(requestSignup());
```

이런식으로 작성하면 조금 더 가독성이 좋을 거 같습니당 ㅎㅎ
그런데 if문을 사용하지 않고 삼항조건연산자를 사용하는 장점이 있을까용??🤔
저도 뭐가 더 좋을 지 모르겠어서 같이 고민해보면 좋을 거 같네요!

```suggestion
    if (isLogin) dispatch(requestLogin());
    dispatch(requestSignup());
```

Reply:
삼항 조건 연산자 표현식이 if문과 다른 차이점은, 삼항 조건 연산자 표현식은 값처럼 사용할 수 있다는 점인데
그래서 어떠한 조건에 따른 값을 결정하는 부분이라 삼항 조건 연산자를 사용해보았네용
그런데 삼항 조건 연산자가 기본적으로 가독성이 좋지 못해서 if문의 사용도 좋을 것  같네요!